service/src/test/java/datawave/microservice/accumulo/lookup/LookupServiceAuditEnabledTest.java

```diff
@@ -144,11 +139,12 @@ public void testLookupAndVerifyLOCALONLY() throws Exception {
     @Test
     public void testAuditTypeACTIVE_VerifyZeroAudits() throws Exception {
         // All mockAuditServer expectation(s) must fail
-        expectedException.expect(AssertionError.class);
-        expectedException.expectMessage(stringContainsInOrder("\n0 request(s) executed"));
         mockDataService.setupMockTable(connector, "tableWithNoAuditRule1");
         mockAuditServer.expect(anything());
-        testLookupAndVerifyAuditUriWithSuccess("tableWithNoAuditRule1", "row3", Auditor.AuditType.ACTIVE);
+        Throwable thrown = Assertions.assertThrows(AssertionError.class, () -> {
```

Comment:
static import for `assertThrows`

Reply:
This will be updated when I merge in Keith's accumulo 2.1 changes.

Reply:
got it